src/components/Footer.tsx

```diff
@@ -16,8 +16,8 @@ export const Footer = () => {
     <Flex
       w="100vw"
       h="40px"
-      position="fixed"
-      bottom="0px"
+      position="absolute"
```

Comment:
I'm not sure if this is what was intended but personally I prefer it that way. The footer just takes up space otherwise and it's not like our pages are so long that it's difficult to reach the bottom to see the footer

Reply:
Yeah this was intended basically for the exact reasons Jayden said. I specifically did it because I didn't like how it would overlap the calendar.

Reply:
Sounds good just wanted to make sure it was behaving how we wanted